### ISSUE TITLE
Deploy Process completed with exit code 1.

### ISSUE THREAD

**Issue by thetwopct** [HubSpot/hubspot-cms-deploy-action#29]
I've been using this Action for several weeks several times a day without issues.

Today it's stopped working and we are unable to deploy.

The warnings from GitHub are:

```
1 error and 3 warnings
[deploy](https://github.com/xxx/actions/runs/7257125293/job/19771049953#step:8:137)
Process completed with exit code 1.

[deploy](https://github.com/xxx/actions/runs/7257125293/job/19771049953)
The following actions uses node12 which is deprecated and will be forced to run on node16: actions/checkout@v2.3.3, actions/setup-node@v1. For more info: https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/

[deploy](https://github.com/xxx/actions/runs/7257125293/job/19771049953#step:2:38)
The `save-state` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

[deploy](https://github.com/xxx/actions/runs/7257125293/job/19771049953#step:2:63)
The `save-state` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

```

Running it again with debug logs on:

```
##[debug]Evaluating condition for step: 'HubSpot Deploy Action'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: HubSpot Deploy Action
##[debug]Loading inputs
##[debug]Evaluating: secrets.hubspot_portal_id
##[debug]Evaluating Index:
##[debug]..Evaluating secrets:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'hubspot_portal_id'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Evaluating: secrets.hubspot_personal_access_key
##[debug]Evaluating Index:
##[debug]..Evaluating secrets:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'hubspot_personal_access_key'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Loading env
Run HubSpot/hubspot-cms-deploy-action@v1.7
##[debug]Evaluating condition for step: 'run'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: run
##[debug]Loading inputs
##[debug]Loading env
Run actions/setup-node@v1
##[debug]isExplicit: 
##[debug]explicit? false
##[debug]isExplicit: 16.20.2
##[debug]explicit? true
##[debug]isExplicit: 18.19.0
##[debug]explicit? true
##[debug]isExplicit: 20.10.0
##[debug]explicit? true
##[debug]evaluating 3 versions
##[debug]matched: 20.10.0
##[debug]checking cache: /opt/hostedtoolcache/node/20.10.0/x64
##[debug]Found tool in cache node 20.10.0 x64
/opt/hostedtoolcache/node/20.10.0/x64/bin/node --version
v20.10.0
/opt/hostedtoolcache/node/20.10.0/x64/bin/npm --version
10.2.3
##[add-matcher]/home/runner/work/_actions/actions/setup-node/v1/.github/tsc.json
##[debug]Added matchers: 'tsc'. Problem matchers scan action output for known warning or error strings and report these inline.
##[add-matcher]/home/runner/work/_actions/actions/setup-node/v1/.github/eslint-stylish.json
##[debug]Added matchers: 'eslint-stylish'. Problem matchers scan action output for known warning or error strings and report these inline.
##[add-matcher]/home/runner/work/_actions/actions/setup-node/v1/.github/eslint-compact.json
##[debug]Added matchers: 'eslint-compact'. Problem matchers scan action output for known warning or error strings and report these inline.
##[debug]Node Action run completed with exit code 0
##[debug]Finished: run
##[debug]Evaluating: inputs.portal_id
##[debug]Evaluating Index:
##[debug]..Evaluating inputs:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'portal_id'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Evaluating: inputs.personal_access_key
##[debug]Evaluating Index:
##[debug]..Evaluating inputs:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'personal_access_key'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Evaluating condition for step: 'run'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: run
##[debug]Loading inputs
##[debug]Evaluating: format('npx --yes --package=@hubspot/cli@latest --call=''hs upload {0} {1} --use-env''
##[debug]', inputs.src_dir, inputs.dest_dir)
##[debug]Evaluating format:
##[debug]..Evaluating String:
##[debug]..=> 'npx --yes --package=@hubspot/cli@latest --call=''hs upload {0} {1} --use-env''
##[debug]'
##[debug]..Evaluating Index:
##[debug]....Evaluating inputs:
##[debug]....=> Object
##[debug]....Evaluating String:
##[debug]....=> 'src_dir'
##[debug]..=> '.'
##[debug]..Evaluating Index:
##[debug]....Evaluating inputs:
##[debug]....=> Object
##[debug]....Evaluating String:
##[debug]....=> 'dest_dir'
##[debug]..=> 'xxx'
##[debug]=> 'npx --yes --package=@hubspot/cli@latest --call=''hs upload . xxx --use-env''
##[debug]'
##[debug]Result: 'npx --yes --package=@hubspot/cli@latest --call=''hs upload . xxx --use-env''
##[debug]'
##[debug]Loading env
Run npx --yes --package=@hubspot/cli@latest --call='hs upload . xxx --use-env'
##[debug]/usr/bin/bash --noprofile --norc -e -o pipefail /home/runner/work/_temp/fed56c66-96eb-44bd-a45c-e9fed20d15c0.sh
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
npm WARN deprecated domexception@2.0.1: Use your platform's native DOMException instead
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated sane@4.1.0: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
A hubspot.config.yml file could not be found
Error:  Cannot destructure property 'auth' of 'getAccountConfig(...)' as it is null.
Error: Process completed with exit code 1.
##[debug]Finished: run
##[debug]Finishing: HubSpot Deploy Action
```

As it has not been working now for several hours, I have tried addressing some of the issues it reported. 

I changed the action to force it to use latest node etc.

```
on:
  push:
    branches:
      - main

jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v4.1.1

      - name: Set up Node.js
        uses: actions/setup-node@v2
        with:
          node-version: '20.x'

      - name: Install HubSpot CLI
        run: npm install -g @hubspot/cli@latest

      - name: HubSpot Deploy Action
        uses: HubSpot/hubspot-cms-deploy-action@v1.7
        with:
          src_dir: .
          dest_dir: my-custom-theme
          portal_id: ${{ secrets.hubspot_portal_id }}
          personal_access_key: ${{ secrets.hubspot_personal_access_key }}
```

But this resulted in the same failure.

`Error:  Cannot destructure property 'auth' of 'getAccountConfig(...)' as it is null.`

Others on Slack are experiencing the same issue:

![Screenshot-2023-12-19 --14 05 31@2x](https://github.com/HubSpot/hubspot-cms-deploy-action/assets/10615884/8cb3f951-be25-4cf8-b193-0bc2648a0e3f)

[HubSpot Status](https://status.hubspot.com) is marked all operational 

Would be good to get some clarity on this issue

Thanks

**Comment by thetwopct** [comment]
Seems to be related to this https://github.com/HubSpot/hubspot-cli/issues/970

**Comment by robbie-hypeanddexter** [comment]
deployments are working for me since the patch in https://github.com/HubSpot/hubspot-cli/issues/970

**Comment by thetwopct** [comment]
Fixed with the 5.1.1 release 
https://github.com/HubSpot/hubspot-cli/releases/tag/v5.1.1
